### PR TITLE
feat: INS-26881 UC Snowflake takeover

### PR DESCRIPTION
**Pull request by chirag-ibm**
Signed-off-by: Chirag Soni <chirag.soni@ibm.com>

### CODE REVIEW

**Comment by chirag-ibm** [comment]
Wicked CLI scan is already executed. The report is available here, https://ibm.box.com/s/u7gqo5wjuyy01f4gk1b2dn90lm343vb8

**Review by ofer-haim — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 infoinsights
```
[the remark this answers is not in the document]
Comment (ofer-haim) [reply filter-plugin/logstash-filter-snowflake-guardium/LICENSE:3]: After consulting with legal department it should be changed to:
```
MIT License
Copyright (c) 2020 infoinsights
© Copyright IBM Corp. 2023 All rights reserved.
```

[the remark this answers is not in the document]
Comment (chirag-ibm) [reply filter-plugin/logstash-filter-snowflake-guardium/LICENSE:3]: @ofer-haim , done. please verify

File: filter-plugin/logstash-filter-snowflake-guardium/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+not work.
+
+
+### Does this work with AWS, Azure, and GCP instances of Snowflake?
+
+Yes. The schema and connection behaviour for Snowflake are the same across all those cloud service
+providers. You may wish to deploy a Guardium collector in the same region as your snowflake instance to
+reduce items such as egress costs.
+
+Have another question? [Open an issue](https://github.com/infoinsights/guardium-snowflake-uc-filter/issues) in this repo and we'll answer and post it here.
```
Comment (johnhaldeman) [inline filter-plugin/logstash-filter-snowflake-guardium/README.md:127]: This suggests people with problems/questions should open an issue with the Information Insights repo? Should they contact support instead?

Comment (johnhaldeman) [inline filter-plugin/logstash-filter-snowflake-guardium/README.md:127]:
```suggestion
Have another question? [Open a support case](https://www.ibm.com/mysupport/s/) with IBM support
```

**Review by johnhaldeman — APPROVED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/README.md (hunk 2 above)
Comment (ofer-haim) [reply filter-plugin/logstash-filter-snowflake-guardium/README.md:127]: Thanks @johnhaldeman,
@chirag-ibm I think we should remove that line, as customers already have support and potential customers should contact sales

File: filter-plugin/logstash-filter-snowflake-guardium/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
```
Comment (RefaelAdi) [inline filter-plugin/logstash-filter-snowflake-guardium/gradle/wrapper/gradle-wrapper.properties:3]: In order to adjust this plug-in need to update the gradle version to 6.5.1 and to make sure all the plug-ins tests are passed

**Review by RefaelAdi — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (chirag-ibm) [reply filter-plugin/logstash-filter-snowflake-guardium/gradle/wrapper/gradle-wrapper.properties]: @RefaelAdi, Done.

**Review by RefaelAdi — APPROVED** [review]

File: filter-plugin/logstash-filter-snowflake-guardium/build.gradle [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+// ===========================================================================
+// plugin info
+// ===========================================================================
+group                      'com.infoinsightsllc.guardium' // must match the package of the main plugin class
+version                    "${file("VERSION").text.trim()}" // read from required VERSION file
+description                = "Guardium Universal Connector Filter Plugin for Snowflake"
+pluginInfo.licenses        = ['Apache-2.0'] // list of SPDX license IDs
+pluginInfo.longDescription = "Guardium Universal Connector Filter Plugin for Snowflake"
+pluginInfo.authors         = ['Information Insights LLC']
```
Comment (PratikshaSonawane) [inline filter-plugin/logstash-filter-snowflake-guardium/build.gradle:15]: change pluginInfo.authors         = ['Information Insights LLC'] to  ['Information Insights LLC', 'IBM']

File: filter-plugin/logstash-filter-snowflake-guardium/build.gradle [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+pluginInfo.email           = ['john.haldeman@infoinsightsllc.com']
```
Comment (PratikshaSonawane) [inline filter-plugin/logstash-filter-snowflake-guardium/build.gradle:16]: Remove this

File: filter-plugin/logstash-filter-snowflake-guardium/build.gradle [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+pluginInfo.homepage        = "http://www.infoinsightsllc.com"
```
Comment (PratikshaSonawane) [inline filter-plugin/logstash-filter-snowflake-guardium/build.gradle:17]: Change this to "https://github.com/IBM/universal-connectors"

**Review by PratikshaSonawane — CHANGES_REQUESTED** [review]

**Review by PratikshaSonawane — APPROVED** [review]

**Pull request merged**